app/steps/expenses_additional_source.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class ExpensesAdditionalSource < Step
+  step_attributes \
+    :dependent_care,
+    :medical,
+    :court_ordered,
+    :tax_deductible
+
+  validates \
```

Comment:
yeah unit test works, too! just wanted to make sure we have *something* testing this validation